readme.md

```diff
@@ -53,7 +70,9 @@ console.log(results);
 ```
 
 Your Answer:
-```text
+```
+I believe that code does not actually return anything because there is no promise attached to it after the find.  Once you find the results, there needs to be a ".then" that actually returns the information.
+In addition to that, if there are more than 1 "Bob"'s, it would not know which one you want to specify
 
```

Comment:
:+1: Fancy! You can, indeed, attach a promise in this situation, You can also use a simple callback, a la:
```
var results = AuthorModel.find({name: "Bob"}, function(results){
  console.log(results);
});
```